config.yaml

```diff
@@ -2,6 +2,12 @@
 # See LICENSE file for licensing details.
 
 options:
+  classic_snap:
```

Comment:
I think we decided that we want to be able to enforce the versions used because grafana-agent can break unexpectedly but I don't recall the exact decision. Leaving this for now, but I will leave #31 open for discussion.